### PR TITLE
Limit orders/submit modal fix

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

This PR fixes the issue with limit-orders submit modal

**Before**
Screenshot from 2022-11-30 14-23-24

**After**
Screenshot from 2022-11-30 14-21-13

### CODE REVIEW

File: src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx [hunk 1]
```diff
@@ -282,13 +282,15 @@ export function L2Content({
   hash,
   pendingText,
   inline,
+  isLimitOrderSubmit,
```
Comment (shoom3301) [inline src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx:285]: I think we shouldn't mix limit-orders into generic `TransactionConfirmationModalMod.tsx` component, would be better to create a new component for this task.
However, for the current release it's ok :)

**Review by shoom3301 — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @nenadV91 , great job!
Maybe some my nitpicks: could we change 'Transaction' to 'Order' and add an arrow to the Explorer link like we do for market orders?
link arrow
market
order
WDYT?

**Comment by nenadV91** [comment]
@elena-zh Added update for those comments

**Comment by elena-zh** [comment]
@nenadV91 , awesome!
Could you please change a bit the link behavior and make it underlined when hover a mouse on it? Currently , it is always underlined

hover

**Comment by nenadV91** [comment]
@elena-zh Fixed

**Review by elena-zh — APPROVED** [review]
Great job!

File: src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx (hunk 1 above)
Comment (anxolin) [reply src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx:285]: agree 100%!

File: src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx [hunk 2]
```diff
@@ -303,6 +305,25 @@ export function L2Content({
 
   const info = getChainInfo(chainId)
 
+  const displayIcon = useMemo(() => {
+    if (isLimitOrderSubmit) {
+      return <AnimatedConfirmation />
+    } else if (!confirmed) {
+      return <CustomLightSpinner src={Circle} alt="loader" size={inline ? '40px' : '90px'} />
```
Comment (anxolin) [inline src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx:312]: Not very good to add PX in the JSX, you should pass the property inline to the component, not the size (I think)

File: src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx [hunk 3]
```diff
@@ -303,6 +305,25 @@ export function L2Content({
...
+    } else {
+      return transactionSuccess ? (
+        <AnimatedConfirmation />
+      ) : (
+        <AlertCircle strokeWidth={1} size={inline ? '40px' : '90px'} color={theme.red1} />
```
Comment (anxolin) [inline src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx:317]: same

**Pull request merged**